nucampsite/src/components/CampsiteInfoComponent.js

```diff
@@ -0,0 +1,52 @@
+import React,{ Component } from "react"
+import { Card, CardImg, CardText, CardBody, CardTitle } from 'reactstrap';
+import {formatDate, toSentenceCase} from "../shared/helpers"
+
+export default class CampsiteInfo extends Component {
+  
+  renderCampsite({image, name, description}) {
```

Comment:
if you like, you can turn `renderCampsite` and the `renderComments` below into arrow functions :)